source/whats-new.txt

```diff
@@ -30,6 +31,31 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _csharp-version-2.27:
+
+What's New in 2.27
+------------------
+
+The 2.27 driver release includes the following new features:
+
+- Adds support for the ``$sample`` aggregation operator.
+
+- Implements the ``Equals()`` method for serializers in LINQ3.
+
+- Includes tests confirming that read and write concerns are not
+  applied to Atlas Search Index Helper commands.
+
+- Disallows the use of the comma character in ``authMechanismProperties``
+  connection string values.
```

Comment:
S: Is there a separate ticket to add this to the OIDC documentation? It should be more clear here also that this is an OIDC-specific change. See Java PR for details: https://github.com/mongodb/docs-java/pull/553

Reply:
I made a ticket to add that info: https://jira.mongodb.org/browse/DOCSP-41035